migrations/v7-MIGRATION.md

```diff
@@ -0,0 +1,41 @@
+# V7 API Migration Guide
+
+This latest release updates the SDK to use BillingClient 6. This version of BillingClient brings an entire new
+subscription model which has resulted in large changes across the entire SDK.
+
+## Updated Code References
+
+This migration guide has detailed class, property, and method changes.
+
+See [Android Native - 5.x to 6.x Migration](https://www.revenuecat.com/docs/android-native-5x-to-6x-migration) for a 
+more thorough explanation of the new Google subscription model announced with BillingClient 5 and how to take advantage of it in V6.
```

Comment:
Do we need this paragraph? Should we just keep it in v6-MIGRATION?

Reply:
I think it doesn't hurt to keep it since some people might be migrating from 5.X and/or will look only at the latest migration guide?